client/capstone-app/src/component/DocInfo.js

```diff
@@ -60,26 +60,39 @@ export default function Doclist() {
   const [name, setName] = useState("");
   const [docData,setDocData]=useState(Docdata())
   const [num, setNum] = useState(0);
+  const[data,setdata] = useState([]);
   
   useEffect(()=>{
     localStorage.setItem("Num", num);
   },)
     
+  useEffect(()=>{
+    fetch("http://localhost:2917/User")
```

Comment:
Instead of adding "http://localhost:2917" in fetch use .evn variable and add it here. Make sure you add .env in .gitignore. Do not push .env file on github.